package.json

```diff
@@ -50,7 +50,7 @@
     "event-stream": "3.3.2",
     "express": "4.13.4",
     "fs-extra": "0.30.0",
-    "irc-framework": "2.5.0",
+    "irc-framework": "git://github.com/metsjeesus/irc-framework.git#encoding",
```

Comment:
I think it's worth waiting for the merge upstream, rather than switching to a fork